azure-pipelines.yml

```diff
@@ -0,0 +1,89 @@
+schedules:
+- cron: "27 3 */1 * *"
+  # 3:27am UTC everyday
+  displayName: Nighthly build
+  branches:
+    include:
+    - master
+    - azure_pipelines_pr_2_ogrisel
+  always: true
+
+jobs:
+  - template: azure/windows.yml
+    parameters:
+      name: windows
+      vmImage: vs2017-win2016
+      matrix:
+        py_3.6_32:
+          PYTHON_VERSION: "3.6.x"
+          PYTHON_ARCH: "x86"
+          NP_BUILD_DEP: "1.12.1"
+        py_3.6_64:
+          PYTHON_VERSION: "3.6.x"
+          NP_BUILD_DEP: "1.12.1"
+        py_3.7_32:
+          PYTHON_VERSION: "3.7.x"
+          PYTHON_ARCH: "x86"
+          NP_BUILD_DEP: "1.14.5"
+          NIGHTLY_BUILD: "true"
+        py_3.7_64:
+          PYTHON_VERSION: "3.7.x"
+          NP_BUILD_DEP: "1.14.5"
+          NIGHTLY_BUILD: "true"
+        py_3.8_32:
+          PYTHON_VERSION: "3.8.x"
+          PYTHON_ARCH: "x86"
+          NP_BUILD_DEP: "1.17.3"
+          SCIPY_BUILD_DEP: "1.4.1"
+          NIGHTLY_BUILD: "true"
+        py_3.8_64:
+          PYTHON_VERSION: "3.8.x"
+          NP_BUILD_DEP: "1.17.3"
+          SCIPY_BUILD_DEP: "1.4.1"
+          NIGHTLY_BUILD: "true"
+
+  - template: azure/posix.yml
+    parameters:
+      name: linux
+      vmImage: ubuntu-16.04
+      matrix:
+        py_3.6_32:
+          MB_PYTHON_VERSION: "3.6"
+          PLAT: "i686"
+        py_3.6_64:
+          MB_PYTHON_VERSION: "3.6"
+        py_3.7_32:
+          MB_PYTHON_VERSION: "3.7"
+          PLAT: "i686"
+          NP_BUILD_DEP: "numpy==1.14.5"
+          NIGHTLY_BUILD: "true"
+        py_3.7_64:
+          MB_PYTHON_VERSION: "3.7"
+          NP_BUILD_DEP: "numpy==1.14.5"
+          NIGHTLY_BUILD: "true"
+        py_3.8_32:
+          MB_PYTHON_VERSION: "3.8"
+          PLAT: "i686"
+          NP_BUILD_DEP: "numpy==1.17.3"
```

Comment:
don't wanna use `oldest-supported-numpy` ?

Reply:
oldest-supported-numpy that also works on Python 3.8.